knowledge_repo/app/templates/auth-login-form.html

```diff
@@ -27,6 +28,10 @@
 .login-form > input[type=submit] {
     margin-bottom: 0px;
 }
+
+button, input, select, textarea {
+    font-weight: 450;
```

Comment:
i don't think `450` has meaning, this should probably be `400` or `500`

Reply:
👍